### PR TITLE
DCOS-50191 Bind I18n object to container

### PR DESCRIPTION
**Pull request by TattdCodeMonkey**
## Testing

This is mainly a refactor so everything should continue to work. Particularly check translations switching and Toasts.

### CODE REVIEW

**Comment by GeorgiSTodorov** [comment]
File conflict.

File: plugins/ui-update/__tests__/notifications-test.ts [hunk 1]
```diff
@@ -10,25 +11,45 @@ jest.mock("../streams", () => ({
...
 
-import * as Notifications from "../notifications";
 import {
   EMPTY_ACTION,
   UIAction,
   UIActions,
   UIActionType
 } from "../types/UIAction";
+import { TYPES } from "#SRC/js/types/containerTypes";
+import { NotificationServiceType } from "@extension-kid/notification-service";
```
Comment (GeorgiSTodorov) [inline plugins/ui-update/__tests__/notifications-test.ts]: Mixing local and non-local imports.

**Comment by nLight** [comment]
Looks like I broke it 👼

**Comment by TattdCodeMonkey** [comment]
@nLight I rebased this and 🤞 CI will be happy

**Pull request merged**